Fabulous.XamarinForms/src/Fabulous.XamarinForms.Controls/Xamarin.Forms.Core.json

```diff
@@ -1628,7 +1628,7 @@
           "name": "UseSafeArea",
           "inputType": "bool",
           "modelType": "bool",
-          "updateCode": "(fun _ _ target -> Xamarin.Forms.PlatformConfiguration.iOSSpecific.Page.SetUseSafeArea((target : Xamarin.Forms.Page).On<Xamarin.Forms.PlatformConfiguration.iOS>(), true) |> ignore)"
+          "updateCode": "updateUseSafeArea"
```

Comment:
It's good that you used ViewConverter for this!
One thing is missing: `"defaultValue": "false"`. without this the default would be true and this is the wrong default as XF defines it.

Reply:
I was a little concerned of setting the default value as it seemed to be set to true in my 0.36 based project (given the issue was I couldn't turn it off) and if I set it to false that would change the layouts of potentially a lot of forms people had already written when they upgraded.

Reply:
Well you are right and it was good that you were concerned but cause we are in a low version this is something that the devs should be aware of. 
If this will happen release 1.0 and higher we should definetly think about this and hint them to sth. like "breaking changes".

Reply:
Starting v1.0, any breaking change will wait for a v2.0 release

>as it seemed to be set to true in my 0.36 based project (given the issue was I couldn't turn it off)
The value was defaulting to true because of this:
https://github.com/fsprojects/Fabulous/blob/bfd59882bcf4a4877cdd62062269a48d6bd4c3e7/Fabulous.XamarinForms/src/Fabulous.XamarinForms/CustomControls.fs#L210-L213

It's still the case, it will still be true by default when we use `View.ContentPage`.
In the json file, the default value is only applied when we remove an attribute we were declaring previously
`View.ContentPage(useSafeArea=true)` -> `View.ContentPage()` = the default value (false) will be applied to "undo" what the use of `useSafeArea` did.